Mage.Sets/src/mage/cards/e/EmergentGrowth.java

```diff
@@ -23,9 +23,7 @@ public EmergentGrowth(UUID ownerId, CardSetInfo setInfo) {
         // Target creature gets +5/+5 until end of turn and must be blocked this turn if able.
         this.getSpellAbility().addTarget(new TargetCreaturePermanent());
         this.getSpellAbility().addEffect(new BoostTargetEffect(5, 5, Duration.EndOfTurn));
-        Effect effect = new MustBeBlockedByAtLeastOneTargetEffect();
-        effect.setText("and must be blocked this turn if able");
```

Comment:
You must use custom text ("and must be blocked this turn if able") instead default one from MustBeBlockedByAtLeastOneTargetEffect.